libs/ledger-live-common/src/families/evm/__tests__/rpc.unit.test.ts

```diff
@@ -1,9 +1,9 @@
 import { AssertionError } from "assert";
-import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import { CryptoCurrency, CryptoCurrencyId } from "@ledgerhq/types-cryptoassets";
 import { DEFAULT_RETRIES_RPC_METHODS, withApi } from "../api/rpc.common";
 
 const fakeCurrency: Partial<CryptoCurrency> = {
-  id: "my_new_chain",
+  id: "my_new_chain" as CryptoCurrencyId,
```

Comment:
Doesn't really make sense here, it's mocked data, and my_new_chain isn't contained in the `@ledgerhq/types-cryptoassets` lib